cmd/crank/beta/validate/image.go

```diff
@@ -332,7 +339,7 @@ func extractLayer(layer conregv1.Layer, destDir string) error { //nolint:gocogni
 // prepareImageReference prepares the image reference by stripping the digest or resolving the tag if necessary.
 func prepareImageReference(image string) (string, error) {
 	if strings.Contains(image, "@") {
-		return strings.SplitN(image, "@", 2)[0], nil
+		return image, nil
```

Comment:
This is a behavior change and `FetchImage` function is not updated. Are you sure you want to return the image with digest? If so, please also update the function comment.